test/source/tests/browser-unit-tests/unit-Sks.js

```diff
@@ -23,7 +23,7 @@
 BROWSER_UNIT_TEST_NAME(`Sks lookup pubkey - trailing slash`);
 (async () => {
   const email = 'john.doe@example.com';
-  const sks = new Sks('https://localhost:8001/');
+  const sks = new Sks(`https://localhost:${MOCK_PORT}/`);
```

Comment:
for unit tests we should leave fixed `:8001`, as different ports are used only in ui tests, so we can run multiple ui tests on different localhost ports